### PR TITLE
[@types/google-apps-script] Add setFixedFooter() to CardBuilder

### LINKED ISSUE #46898: [@types/google-apps-script] CardService is missing methods
Recently I've been trying to add a footer, if you try to do the suggested cat tutorial idea: https://developers.google.com/gsuite/add-ons/cats-quickstart, it cannot find `setFixedFooter()`

area of concern:
```
...

// Create a footer to be shown at the bottom.
  var footer = CardService.newFixedFooter()
      .setPrimaryButton(CardService.newTextButton()
          .setText('Powered by cataas.com')
          .setOpenLink(CardService.newOpenLink()
              .setUrl('https://cataas.com')));

  // Assemble the widgets and return the card.
  var section = CardService.newCardSection()
      .addWidget(image)
      .addWidget(buttonSet);
  var card = CardService.newCardBuilder()
      .addSection(section)
      .setDisplayStyle(DisplayStyle.REPLACE) // cannot find setDisplayStyle and DisplayStyle enum
      .setFixedFooter(footer); // cannot find
...

```

note: on version:     "@types/google-apps-script": "^1.0.14"

@PopGoesTheWza , @grant

### PR DESCRIPTION
**Pull request by jessety**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

Fixes #46898

### CODE REVIEW

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 1]
```diff
@@ -152,11 +152,13 @@ declare namespace GoogleAppsScript {
     }
     /**
      * A builder for Card objects.
+     * https://developers.google.com/apps-script/reference/card-service/card-builder
      */
     interface CardBuilder {
       addCardAction(cardAction: CardAction): CardBuilder;
       addSection(section: CardSection): CardBuilder;
       build(): Card;
+      setFixedFooter(footer: FixedFooter): CardBuilder;
```
Comment (mtgto) [inline types/google-apps-script/google-apps-script.card-service.d.ts:161]:
```suggestion
      setFixedFooter(fixedFooter: FixedFooter): CardBuilder;
```
https://developers.google.com/apps-script/reference/card-service/card-builder#setFixedFooter(FixedFooter)

Comment (mtgto) [reply types/google-apps-script/google-apps-script.card-service.d.ts:161]: It seems good.
@jessety Can you fix it before I approve?

Comment (jessety) [reply types/google-apps-script/google-apps-script.card-service.d.ts:161]: Nice catch, thanks @mtgto!

**Comment by mtgto** [comment]
@jessety Sorry to keep you waiting. Your changes is good.
But #46898 is already fixed in #51952 🙇  (I know it's newer than you created this pull request)

You can close this pull request. Thank you for your contribution.

**Pull request closed without merging**